specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2018-07-01/storagesync.json

```diff
@@ -2595,7 +2595,7 @@
             "type": "object",
             "description": "Trigger Rollover Request.",
             "properties": {
-                "certificateData": {
+                "serverCertificate": {
```

Comment:
@ankushbindlish2 why is a stable version of the service being changed here. This is a breaking change on the service side and we should undo this change